flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/drivers/parameter/Parameter.java

```diff
@@ -40,6 +40,15 @@
 	String getUsage();
 
 	/**
+	 * A hidden parameter is parsed from the command-line configuration but is
+	 * not printed in the usage string. This can be used for power-user options
+	 * not displayed to the general user.
```

Comment:
The initial use case was for disabling object reuse which a user should not be concerned with (all the algorithms support object reuse) or even know about but is used by the integration tests to validate the algorithms under object reuse and for comparison benchmarking.

Another example is the next PR which hides the "algorithm" parallelism which is only needed for large executions and might be confusing with the `-p` "job" parallelism.